src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/ConvertToJsonCommand.cs

```diff
@@ -683,5 +426,21 @@ private object ProcessCustomObject<T>(object o, int depth)
             }
             return result;
         }
+
+        /// <summary>
+        /// Check if Stopping is set and throw special exception to stop.
+        /// </summary>
+        private void CheckStopping()
```

Comment:
Seems we should remove the the help method - used one time.

Reply:
Yes, with the removal of the unused code, this helper is no longer needed.  Will remove.